<!-- launch file -->

<launch>

	<node pkg="mavlink_lora" type="mavlink_lora_node" name="mavlink_lora" respawn="true" output="screen">
		<param name="serial_device" value="/dev/ttyUSB1"/>
		<param name="serial_baudrate" value="115200"/>
	</node>

	<node pkg="uav_restriction" type="uav_restriction_node.py" name="uav_restriction_node" respawn="false" output="screen">
		<param name="serial_device" value="/dev/ttyUSB0"/>
		<param name="serial_baudrate" value="115200"/>
		<param name="emulate_arduino_send" value="0"/>
	</node>

</launch>
